dev-libs/jansson/jansson-2.12.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Foundation
```

Comment:
This is wrong, please use repoman to commit to fix it automatically for you.

Reply:
repoman doesn't complain about this. Are you expecting `Gentoo Authors` instead of `Gentoo Foundation`?

Reply:
Yes. There's no reason to 'complain' about it since then +90 % of the ebuilds in the tree are 'faulty', but if you use `repoman commit` it updates the copyright header for you automatically as should be done for every new commit. You can also now manually update it too of course :P

Reply:
Fixed